### DISCUSSION TITLE
Totalling Tables

### DISCUSSION THREAD

**Discussion by HeronCS** [Budibase/budibase#5729] [Help] [answered]
Hi, wondering if it's an already implemented feature or to be requested.

Having totals at the end of numbered columns in a table.

Need this done ASAP, could someone point me to how to do this, for now.
Whether it's supported or not.

**Comment by gerhardmpl** [comment]
I think it is not a feature of the table component itself, but using a view and a separate sum row might get you there. Check #3071 for a related discussion.

**Comment by HeronCS** [comment, marked as answer]
Please see #6278
